File: laramie-tesoreria-webapi/src/domain/entities/observacion.ts
export default class Observacion {

    id: number;
	entidad: string;
	idEntidad: number;
	detalle: string;
	idUsuario: number;
	fecha: Date;

	constructor(
        id: number = 0,
		entidad: string = "",
		idEntidad: number = 0,
		detalle: string = "",
		idUsuario: number = 0,
		fecha: Date = null
	)
	{
        this.id = id;
		this.entidad = entidad;
		this.idEntidad = idEntidad;
		this.detalle = detalle;
		this.idUsuario = idUsuario;
		this.fecha = fecha;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.entidad = row.entidad ?? "";
		this.idEntidad = row.idEntidad ?? 0;
		this.detalle = row.detalle ?? "";
		this.idUsuario = row.idUsuario ?? 0;
		this.fecha = row.fecha ?? null;
	}

}

File: modules/tesoreria/models/observacion.py
from sqlalchemy import Column, BigInteger, String, DateTime

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Observacion(Base):
    __tablename__ = "tesoreria_observaciones"

    id = Column(BigInteger, primary_key=True)
    entidad = Column(String(50), nullable=False, index=True)
    id_entidad = Column(BigInteger, nullable=False, index=True)
    detalle = Column(String(1000), nullable=False)
    id_usuario = Column(BigInteger, nullable=True)
    fecha = Column(DateTime, nullable=False)
